docs/course_materials/exercises/test_exercises/README_run_tests.md

```diff
@@ -0,0 +1,22 @@
+# Checking Exercises
```

Comment:
I think that maybe this README can be inside ROSE folder because it will be easier for the one who will run these scripts where to start.